tensorboard/plugins/npmi/npy_to_embedding_data_demo.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Adds embedding representations to the annotations of a dataset to enable
+similarity-based analysis.
+
+To use this, provide a .npy file containing embeddings, which will be converted
+to a logfile alongside other logs for this run.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from absl import app
+from absl import flags
+from tensorboard.plugins.npmi import summary
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    "out_path", None, "Directory to write the new log file to.",
+)
+flags.DEFINE_string(
+    "embeddings_path",
+    None,
+    "Path to the numpy .npy file containing the embeddings. Embeddings can "
```

Comment:
Could you be slightly more precise about the expected contents of the file here?  Based on the summary function, it accepts a float tensor of shape `(num_annotations, embedding_dimension)` so it would be nice to repeat that here.